modules/generic_worker/templates/run-generic-worker.sh.erb

```diff
@@ -13,6 +13,17 @@ rm -f tasks-resolved-count.txt
 export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
 source /usr/local/share/generic-worker/bugzilla-utils.sh
 
+. /etc/rc.common
```

Comment:
I don't think you need to do this network check because puppet already waits for network before running and creating the semaphore file that triggers this.

Also, we aren't controlling rc.common. So this *could* do anything.

Reply:
Please remove this network check block.

Reply:
Thanks! that was causing me stress ;)